src/mock/serde/mod.rs

```diff
@@ -0,0 +1,45 @@
+pub mod json_string {
```

Comment:
Why do we define this module ?
It seems that it is not used currently ?

Reply:
It's used in `mock/model.rs`  : 
```rust
#[derive(Debug, Serialize, Deserialize, PartialEq, Clone)]
pub enum RequestBody {
    JSON {
        #[serde(with = "serde_mock::json_string")]
        json: Value,
    },
    TEXT {
        text: String,
    },
}
```